tests/checks/expansion.fish

```diff
@@ -337,11 +341,11 @@ printf '<%s>\n' ($fish -c 'echo "$abc["' 2>&1)
 
 set -l pager command less
 echo foo | $pager
-#CHECKERR: {{.*}}checks/expansion.fish (line 339): The expanded command is a keyword.
+#CHECKERR: {{.*}}checks/expansion.fish (line {{\d+}}): The expanded command is a keyword.
 #CHECKERR: echo foo | $pager
 #CHECKERR:            ^~~~~^
 
-"command" -h
+command -h
```

Comment:
(this one was quoted because that neutralizes `command` as a keyword, deferring to the builtin always. it's unnecessary since that happens for `command -h` anyway)